### PR TITLE
Fix PEP8 code style and misspelling issues

### PR DESCRIPTION
**Pull request by vxfield**
This PR fixes hundreds of PEP8/Pylint and some misspelling issues in preparation #50 (enforce code style guidelines in the CI/CD pipeline)

### CODE REVIEW

File: azure-quantum/azure/quantum/job.py [hunk 1]
```diff
@@ -50,40 +52,50 @@ def has_completed(self):
...
-            # output_data_uri does not contains SAS token, get sas url from service
-            blob_client = BlobClient.from_blob_url(self.details.output_data_uri)
-            blob_uri = self.workspace._get_linked_storage_sas_uri(blob_client.container_name, blob_client.blob_name)
+            # output_data_uri does not contains SAS token,
+            # get sas url from service
+            blob_client = BlobClient.from_blob_url(
+                self.details.output_data_uri)
+            blob_uri = self.workspace._get_linked_storage_sas_uri(
+                blob_client.container_name, blob_client.blob_name)
             payload = download_blob(blob_uri)
```
Comment (sanjgupt) [inline azure-quantum/azure/quantum/job.py:91]: I think we need import download_blob from azure.storage.blob for this code path to work.

**Comment by vxfield** [comment]
I still need to do more validations and re-run all tests, but at least most of applicable PEP8 issues were fixe.

I tried to be consistent on line breaks, but I may have missed a few things.

There were some exceptions that we can discuss, specially regarding long lines that contains URLs.

**Review by sanjgupt — COMMENTED** [review]
I have taken a quick look but I will go through it again, later today and run the tests as well.

File: azure-quantum/azure/quantum/job.py (hunk 1 above)
Comment (sanjgupt) [reply azure-quantum/azure/quantum/job.py:91]: I think we should run pycodestyle on this repo and it will point out pep8 style issues as well.

Comment (sanjgupt) [reply azure-quantum/azure/quantum/job.py:91]: To autofix issues 'black' is a tool to help. It isn;t 100% pep8 compliant but there is way around it

Comment (vxfield) [reply azure-quantum/azure/quantum/job.py:91]: Yes, we have another issue #50 that will address that.
For now, I manually ran and fixed the issues using pylint and flake8.

Comment (vxfield) [reply azure-quantum/azure/quantum/job.py:91]: Did you find a specific style issue with the highlighted code?
Is there a better way to break those lines so they will be under 80 chars?

**Comment by vxfield** [comment]
Issue #50 that will address running linting tools on the repo with the CI/CD pipeline

**Comment by vxfield** [comment]
I ran the unit tests and code is running fine.
I also fixed a credential scan false positive.
This is now ready for final review + merge.

**Review by sanjgupt — APPROVED** [review]

**Pull request merged**